src/zcbor_decode.c

```diff
@@ -964,48 +964,32 @@ bool zcbor_any_skip(zcbor_state_t *state, void *result)
 	INITIAL_CHECKS();
 	zcbor_major_type_t major_type = MAJOR_TYPE(*state->payload);
 	uint8_t additional = ADDITIONAL(*state->payload);
-	size_t value;
-	uint_fast32_t num_decode;
-	uint_fast32_t temp_elem_count;
-	uint_fast32_t elem_count_bak = state->elem_count;
-	uint8_t const *payload_bak = state->payload;
-	uint64_t tag_dummy;
-
-	payload_bak = state->payload;
-
-	if (!zcbor_multi_decode(0, ZCBOR_LARGE_ELEM_COUNT, &num_decode,
-			(zcbor_decoder_t *)zcbor_tag_decode, state,
-			(void *)&tag_dummy, 0)) {
-		state->elem_count = elem_count_bak;
-		state->payload = payload_bak;
-		ZCBOR_FAIL();
-	}
+	uint64_t value;
+	uint32_t tag_dummy;
```

Comment:
nit: What about defining `tag_dummy` in the scope of `while` loop, where it's used?

Reply:
Done